x-pack/plugin/snapshot-repo-test-kit/qa/azure/src/javaRestTest/java/org/elasticsearch/repositories/blobstore/testkit/AzureSnapshotRepoTestKitIT.java

```diff
@@ -99,4 +113,8 @@ protected Settings repositorySettings() {
 
         return Settings.builder().put("client", "repository_test_kit").put("container", container).put("base_path", basePath).build();
     }
+
+    private static boolean notNullOrEmpty(String s) {
+        return s != null && false == s.isEmpty();
```

Comment:
Why not `Strings.hasText`?

Reply:
Strings.hasText makes perfect sense here 👍